docs/src/pages/[platform]/getting-started/troubleshooting/troubleshooting.react.mdx

```diff
@@ -160,6 +160,18 @@ This is a known problem when using the `jsdom` library (a dependency of Jest) wi
 
 Next.js 13.4+ introduces [app directory](https://nextjs.org/docs/app/building-your-application/routing#the-app-directory) with the usage of [Server Components](https://nextjs.org/docs/getting-started/react-essentials#server-components). To use Amplify UI componenents, you must use them inside the Client Component tree by placing `"use client"` at the boundary between your Client and Server component module graph. Please see the Next.js [documentation](https://nextjs.org/docs/getting-started/react-essentials#third-party-packages) on how you could achieve this.
 
+### Next 13.4+: `Module not found errors`
```

Comment:
Is the console error output `Module not found errors`?

Reply:
+1, let's add the specific error message here

Reply:
This is what terminal will output

Reply:
It should say 
```suggestion
### Next 13.4+: `Module not found` Errors
```

based on what I see below, correct?

```
Module not found: Can't resolve 'encoding' in 'xxx'
```

Reply:
This is what i am seeing:
```
./node_modules/node-fetch/lib/index.js
Module not found: Can't resolve 'encoding' in '/Users/cpollman/Amplify/use-auth-examples/node_modules/node-fetch/lib'
```

Reply:
`Module not found Errors` makes sense to me

Reply:
IMO we should separate "errors" out from the code highlight (@wlee221 suggestion)

Reply:
Just meant the heading (taken from the suggestion @wlee221 had):
```suggestion
### Next 13.4+: `Module not found` Errors
```

Reply:
Looking around other headings. Kinda make sense to separate out

```
Next 13.4+: Can't resolve 'encoding' in 'xxx' and Module not found: Can't resolve 'aws-crt' in 'xxx'
```

Reply:
+1, we should include the errors separately to allow for better SEO findability for customers

Reply:
Synced offline. We are aligned to use the shorter version